cg-spec/editors_draft.html

```diff
@@ -2229,38 +2238,33 @@ <h2>Merging RDF-star graphs</h2>
 
-        <p>This makes it impossible for RDF-star-aware systems to reliably exchange <a>RDF-star graphs</a> in their mapped form using non-RDF-star concrete syntaxes (unless of course the <a>RDF-star graph</a> contains no <a>embedded triple</a>). However, such systems can always use <a href="#turtle-star">Turtle-star</a> or other extended concrete syntaxes, so that does not limit interoperability among them. On the other hand, it prevents the unrestricted use of the special properties, because that may lead to surprising corner cases, as illustrated in <a href="#unrestricted-use-of-ST-and-SS"></a>. Supporting these corner cases would be a significant burden on RDF-star implementations, for a very limited utility.</p>
+        <p>In particular, if |G| and |H| contain the same <a>embedded triple</a>, this triple will be mapped to a single blank node in |unstar|(|G| ∪ |H|), but in two potentially different blank nodes in |unstar|(|G|) ∪ |unstar|(|H|). These blank nodes will need to be unified in order to reconstruct |unstar|(|G| ∪ |H|).</p>
 
-        <p>Furthermore, it is expected that some implementations will not rely on the mapping, but represent and work directly with the <a href="#concepts">abstract syntax</a> of RDF-star. For these implementations, having to handle both the native and the mapped representation of <a>embedded triples</a> would be even more challenging.</p>
+        <p>Conversely, nothing in the definition of |unstar| prevents it from generating the same blank node |b| for an <a>embedded triple</a> of |G|, and for a <em>different</em> <a>embedded triple</a> of |H| (provided that |b| is not present in either |G| nor |H|). In that case, not only would |unstar|(|G|) ∪ |unstar|(|H|) be different from |unstar|(|G| ∪ |H|), but it would lose some information by mixing the descriptions of the two <a>embedded triples</a>.</p>
 
-        <pre id="unrestricted-use-of-ST-and-SS"
-        data-transform="updateExample"
-        data-content-type="text/x-turtle-star"
-        class="nohighlight example"
-        >
-          <!--
-          :alice :says << :bob :age 42 >>.
-          << :bob :age 42 >> :ST :charlie;
-                             :SS "<http://example.org/charlie>".
+      </section>
 
-          # assuming that :ST and :SS stand for the corresponding special IRIs,
-          # the graph above entails the graph below
+      <section>
+        <h2>Other uses of the |unstar| mapping</h2>
 
-          :alice :says << :charlie :age 42 >>.
-          -->
-        </pre>
+        <p>Since the |unstar| mapping defined <a href="#mapping">above</a> transforms any <a>RDF-star graph</a> into a standard ("non-star") <a>RDF graph</a>, it might be tempting to use this mapping for conveying <a>RDF-star graphs</a> to legacy RDF systems. This however has a number of caveats.</p>
```

Comment:
```suggestion
        <p>Since the |unstar| mapping defined <a href="#mapping">above</a> transforms any <a>RDF-star graph</a> into a standard ("non-star") <a>RDF graph</a>, it might be tempting to use this mapping for conveying <a>RDF-star graphs</a> to legacy RDF systems. However, this has a number of caveats.</p>
```